lib/src/st_tx_video_session.c

```diff
@@ -2102,6 +2109,7 @@ static int tv_attach(struct st_main_impl* impl, struct st_tx_video_sessions_mgr*
     s->st20_frame_size = s->st22_codestream_size + s->st22_box_hdr_length;
```

Comment:
Should we assign st20_fb_size for st22_frame_ops case also?

Reply:
Yes, also missed the assignment in rx. Running gtest again.